docs/respondent_management_to_electronic_questionnaire.rst

```diff
@@ -75,12 +58,26 @@ The runner can optionally accept the following keys.
   A URL for a remote survey JSON. This claim is used to tell Survey Runner to load the schema JSON from a remote location
 ``case_ref``
   The case reference identified by the above UUID (e.g. "1000000000000001")
+``case_type``
+  The type of case
+``account_service_url``
+  The url of the account service used to launch the survey
 ``account_service_log_out_url``
   The logout url of the account service used to launch the survey.  Not required for services that don't have a log in function (i.e., respondent home)
+``channel``
+  The channel (client) from which the questionnaire was launched
+
+Schema Metadata
```

Comment:
The other headings refer to fields rather than metadata, and it would be clearer if it was more explicit, something like:
```suggestion
Schema Defined Required Fields
```